lib/node/nodes/merge.js

```diff
@@ -54,10 +54,12 @@ Merge.prototype.sql = function() {
 Merge.prototype._getColumns = function() {
     var geomColumn = ((this.source_geometry === 'left_source') ? INPUT_LEFT_ALIAS : INPUT_RIGHT_ALIAS) +
         '.the_geom as the_geom';
+    var geomWmColumn = ((this.source_geometry === 'left_source') ? INPUT_LEFT_ALIAS : INPUT_RIGHT_ALIAS) +
+        '.the_geom_webmercator as the_geom_webmercator';
```

Comment:
What happens if one of the sources is another analysis and `the_geom_webmercator` does not exist?